index.html

```diff
@@ -33,11 +33,179 @@ <h2 id="id-my-name">Marc Raphael</h2>
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+
+          <!--project five-->
+          <li class="class-project">
+            <img src="./images/Rectangle21(5).png" alt="project image" class="class-project-image">
+            <div class="class-project-description">
+              <ul class="class-project-languages">
+                <li class="class-project-name">Project name goes here</li>
+                <li class="class-project-li">HTML/CSS</li>
+                <li class="class-project-li">Ruby on rails</li>
+                <li class="class-project-li">Javascript</li>
+              </ul>
+              <a href="#" target="_parent" class="class-project-link">
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+          
+          <!--project six-->
+          <li class="class-project">
+            <img src="./images/Rectangle21(6).png" alt="project image" class="class-project-image">
+            <div class="class-project-description">
+              <ul class="class-project-languages">
+                <li class="class-project-name">Project name goes here</li>
+                <li class="class-project-li">HTML/CSS</li>
+                <li class="class-project-li">Ruby on rails</li>
+                <li class="class-project-li">Javascript</li>
+              </ul>
+              <a href="#" target="_parent" class="class-project-link">
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+        </ul>
+      </section>
+
+      <!--The section about my self-->
+      <section id="id-section-about-my-self">
+
+        <!--Introduction to the the project list-->
+        <h1 class="class-sections-titles">About</h1>
+
+        <!--litle description about me-->
+        <p id="id-about-description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+
+        <!--Button to the CV-->
+        <h2 id="id-about-cv">
+          <span id="id-about-span">Get my CV</span>
+          <img src="./images/ic_download.svg" alt="union pic" id="id-about-pic">
+        </h2>
+
+        <!--Languages used-->
+        <div class="class-about-div" id="id-about-languages">
+          <ul class="class-about-div-ul">
+            <li class="class-about-div-title"><span>Languages</span></li>
```

Comment:
- [ ] The `Languages` here is not one of the `li` items here. Please take it out from the `ul` and make it a heading with `hx' tag. Apply this change to the other categories too.